presto-orc/src/test/java/com/facebook/presto/orc/BenchmarkZstdJniDecompression.java

```diff
@@ -0,0 +1,284 @@
+
+            size += xBlock.getPositionCount();
+            size += yBlock.getPositionCount();
+            size += zBlock.getPositionCount();
+            for (int position = 0; position < batchSize; position++) {
+                DOUBLE.getDouble(xBlock, position);
+                BIGINT.getLong(yBlock, position);
+                charType.getSlice(zBlock, position);
+                arrayType.getObject(aBlock, position);
+            }
+        }
+
+        assertEquals(rows, reader.getFooter().getNumberOfRows());
+        assertEquals(size, 263598);
+    }
+
+    @Benchmark
+    public void decompressJni()
+            throws OrcCorruptionException
+    {
+        decompressList(jniDecompressor);
+    }
+
+    @Benchmark
+    public void decompressJava()
+            throws OrcCorruptionException
+    {
+        decompressList(orcJavaDecompressor);
+    }
+
+    private void decompressList(OrcDecompressor decompressor)
+            throws OrcCorruptionException
+    {
+        for (Unit unit : list) {
+            int outputSize = decompressor.decompress(unit.compressedBytes, 0, unit.compressedLen, new OrcDecompressor.OutputBuffer()
+            {
+                @Override
+                public byte[] initialize(int size)
+                {
+                    return decompressedBytes;
+                }
+
+                @Override
+                public byte[] grow(int size)
+                {
+                    throw new RuntimeException();
+                }
+            });
+            assertEquals(outputSize, unit.srcLen);
+        }
+    }
+
+    private static List<Unit> generateWorkload()
+    {
+        List<Unit> list = new ArrayList<>();
+        for (int i = 0; i < 10; i++) {
+            byte[] srcBytes = getAlphaNumericString(srcLen).getBytes();
+            byte[] compressedBytes = new byte[srcLen * 32];
+            int size = compressor.compress(srcBytes, 0, srcBytes.length, compressedBytes, 0, compressedBytes.length);
+            list.add(new Unit(srcBytes, srcLen, compressedBytes, size));
+            System.out.println(srcLen + " : " + size);
+        }
+        return list;
+    }
+
+    static String getAlphaNumericString(int n)
+    {
+        // chose a Character random from this String
+        String alphaNumericString = "USINDIA";
+
+        // create StringBuffer size of AlphaNumericString
+        StringBuilder sb = new StringBuilder(n);
+
+        for (int i = 0; i < n; i++) {
+            // generate a random number between
+            // 0 to AlphaNumericString variable length
+            int index
+                    = (int) (alphaNumericString.length()
+                    * Math.random());
+
+            // add Character one by one in end of sb
+            sb.append(alphaNumericString
+                    .charAt(index));
+        }
+
+        return sb.toString();
+    }
+
+    static class Unit
+    {
+        byte[] srcBytes;
```

Comment:
Could you follow our developer guideline (https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#formatting) and clean up this patch including:
- Spell out every word (e.g., src -> source)
- Provide meaningful names to variables (e.g., i -> length)
- We don't use ArrayList, LinkedList, and other Java native containers unless necessary (e.g., mutability or nulls). Instead, we use Guava ImmutableSet/Map/List